mandatory/1-syntax-errors.js

```diff
@@ -4,13 +4,13 @@ function addNumbers(a b c) {
     return a + b + c;
 }
 
-function introduceMe(name, age)
+function introduceMe(name, age) {
 return "Hello, my name is " + name "and I am " age + "years old";
-
+}
 function getTotal(a, b) {
-    total = a ++ b;
+    total = a + b;
```

Comment:
avoid declaring global variables like this https://stackoverflow.com/questions/10525582/why-are-global-variables-considered-bad-practice